libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSDKWindowManagerTests.m

```diff
@@ -25,6 +25,7 @@
 
 #import <XCTest/XCTest.h>
 #import "SFSDKWindowManager.h"
+#import "SFApplicationHelper.h"
 
 @interface SFSDKWindowManagerTests: XCTestCase{
```

Comment:
I'm still looking at ways to unit test with multiple scenes but for now added tests for the new APIs for single scenes and manually tested multiple scenes